### PR TITLE
feat(UI): Add Hero component

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?
This PR is part of the Component Placement project, to bring some components from our starters (gatsby.store/nextjs.store).

image

## How it works? 
The `Hero` component works together with its own wrapper components for each content's details:

`HeroImage`: the wrapper component for the hero's image;
`HeroHeading`: the wrapper component for the content (title, description and call to action link).

## How to test it?
Unit tests through Jest: `yarn test packages/ui/src/organisms/Hero/Hero.test.tsx`

## References
[Component Placement - FSSS-351](https://vtex-dev.atlassian.net/browse/FSSS-351?atlOrigin=eyJpIjoiMmI0OTJlZDI3MmU5NGE4NTlmNGIxNGY5YTc3NDZjMWMiLCJwIjoiaiJ9)

### CODE REVIEW

File: packages/ui/src/organisms/Hero/Hero.test.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    it('`HeroHeading` component should have `data-hero-heading` attribute', () => {
+      expect(heroHeading).toHaveAttribute('data-hero-heading')
+    })
+  })
+
+  describe('Accessibility', () => {
+    it('should have no violations', async () => {
+      const { getByTestId } = render(<HeroTest />)
+
+      expect(await axe(getByTestId('store-hero'))).toHaveNoViolations()
```
Comment (eduardoformiga) [inline packages/ui/src/organisms/Hero/Hero.test.tsx:56]:
```suggestion
      expect(await axe(getByTestId('store-hero'))).toHaveNoViolations()
      expect(await axe(getByTestId('store-hero'))).toHaveNoIncompletes()
```

**Review by eduardoformiga — APPROVED** [review]
Great Job! ✨ 

Do we have the starters' PRs already?

**Comment by hellofanny** [comment]
> Great Job! ✨
> 
> Do we have the starters' PRs already?

Thanks for reviewing! Just one starter PR in draft. I'll open it soonn ~

File: packages/ui/src/organisms/Hero/Hero.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+
+export interface HeroProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  children: React.ReactNode
```
Comment (filipewl) [inline packages/ui/src/organisms/Hero/Hero.tsx:10]: It seems we can skip this! Both `HeroHeading` and `HeroImage` don't have/need it as well.
```suggestion
```

File: packages/ui/src/organisms/Hero/HeroHeading.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+export interface HeroHeadingProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const HeroHeading = forwardRef<HTMLDivElement, HeroHeadingProps>(
+  function BannerContent(
```
Comment (filipewl) [inline packages/ui/src/organisms/Hero/HeroHeading.tsx:12]:
```suggestion
  function HeroHeading(
```

**Review by filipewl — APPROVED** [review]

File: packages/ui/src/organisms/Hero/HeroHeading.tsx (hunk 3 above)
Comment (hellofanny) [reply packages/ui/src/organisms/Hero/HeroHeading.tsx:12]: oopsssss

**Pull request merged**